doc/ch-image_desc.rst

```diff
@@ -534,6 +555,50 @@ Same, except place the image in :code:`/tmp/buster`::
    bin   dev  home  lib64  mnt  proc  run   srv  tmp  var
    boot  etc  lib   media  opt  root  sbin  sys  usr
 
+Print list of targetable architecturess for the Debian Buster image.::
+
+   $ ch-image pull --list-arch debian:buster
+   fat manifest: using existing
+   available platforms:
+   amd64
+   arm/v5
+   arm/v7
+   arm64/v8
+   386
+   mips64le
+   ppc64le
+   s390x
+
+Specify the aarch64 version of Debian Stretch using :code:`--arch`.::
+
+   $ ch-grow pull --arch=arm64/v8 debian:buster
+   pulling image:   debian:buster
+   fat manifest: using existing
+   architecture: arm64v8 (aarch64)
+   manifest: downloading
+   layer 1/1: 04aacb1: downloading
+   layer 1/1: 04aacb1: listing
+   validating tarball members
+   resolving whiteouts
+   flattening image
+   layer 1/1: 04aacb1: extracting
+   done
+
+Specify the aarch32 version of Debian Stretch using long name.::
```

Comment:
Would anyone want to do this?

Reply:
Do what exactly? Pull a aarch32 image or pull something different than the host they're on?